test/src/test/java/org/corfudb/infrastructure/ReconfigurationEventHandlerTest.java

```diff
@@ -58,7 +58,7 @@ public void updateLayoutOnFailure() {
         failedServers.add(getEndpoint(SERVERS.PORT_2));
```

Comment:
Would it be possible to remove SERVERS.PORT_0 instead of 2.
This way we can test 2 more items in this test with the given policy:
1. Sequencer rollover.
2. LogUnit removal.